database/postgres/dal/jwt.py

```diff
@@ -0,0 +1,84 @@
+from sqlalchemy.orm import Session
+from database.models.token_blocklist import TokenBlocklist
+from database.postgres.dto.jwt import jwtDTO
+from sqlalchemy.exc import SQLAlchemyError
+from typing import Optional, List
+from datetime import datetime
+
+class jwtDAL:
```

Comment:
```suggestion
class JwtDAL:
```

But maybe we could rename it to something better. JWT is just one type of access tokens. It's now wise to have our whole DAL depend on that. I'd suggest trying some more generic name here.

AccessTokensDAL maybe? IDK